### DISCUSSION TITLE
List-KR 필터 질문

### DISCUSSION THREAD

**Discussion by Maccvc** [List-KR/community#3] [Q&A]
macOS와 iOS 환경에서 Adguard를 최대한 가볍게 쓰고 싶은데

List-KR 안내를 보니
'AdGuard 플렛폼에서는 별도의 안내가 없는 경우 **AdGuard 베이스 필터**와 함께 사용하셔야 최상의 결과를 얻으실 수 있습니다.
또 AdGuard에서 제공하는 **추적 보호 필터**, **URL 추적 필터** 그리고 **방해 요소 필터** 또한 함께 사용하면 
웹 사이트에 설치된 추적기나 우클릭 방지 스크립트도 효율적으로 차단할 수 있습니다.'

라고 되어있는데

**Adguard 베이스 필터, 추적 보호 필터, URL 추적 필터, 방해 요소 필터**를 
**Adguard DNS 필터** 하나로 대체해도 문제가 없나요?

**Comment by piquark6046** [comment]
한국어권 웹 사이트만 방문하신다면 문제가 없을거라고 생각합니다.
예를 들어서, YouTube 광고를 차단하시고 싶으시면, AdGuard 베이스 필터에 있는 scriptlet 규칙이 필요한데 이러한 규칙은 DNS 필터에 포함되어 있지 않습니다.

**Reply by Maccvc** [reply]
유튜브는 프리미엄 사용중이라 상관없지만
같은 방식으로 scriptlet 규칙이 필요한 것들은 DNS 필터로 대체할 수 없겠군요
그럼 베이스 필터와 list-kr을 쓰는게 가장 효율적이겠네요

답변 감사합니다!!
